app/src/main/res/values/strings.xml

```diff
@@ -129,14 +129,30 @@
     <string name="data_content_info">Here is the history of contacts with other users of eRouška. It doesn\'t indicate their health status.</string>
     <string name="help_write_chabot">Chat with Anežka – eRouška support</string>
     <string name="contacts_open_chatbot">Anežka – eRouška support</string>
+    <string name="risky_contacts_title">Risky contacts</string>
     <string name="privacy_header_text">We value your privacy, sharing data is under your control</string>
     <string name="privacy_body_text_1">eRouška doesn\'t have any personal information about you and collects only anonymous data about other eRouška applications in your vicinity.</string>
     <string name="privacy_body_text_2"><![CDATA[<a href="%1$s">Multiple independent audits</a> guarantee that the application doesn\'t share your data without your consent. Only you can share your collected data when administration of hygienic station requests them.]]></string>
-<!--    <a href="%1$s">Několik nezávislých auditů</a> garantuje, že aplikace neodesílá vaše osobní údaje bez vašeho vědomí. Odešlete je vy sami a pouze hygienikům.-->
     <string name="privacy_finish">Finish activation</string>
     <string name="privacy_toolbar_title">Privacy</string>
     <string name="activation_error_title">Error</string>
     <string name="activation_error_header">Activation couldn\'t be finished</string>
     <string name="activation_error_body">Check yout internet connection and try again to activate your application. If problems persist, contact us at info@erouska.cz</string>
     <string name="activation_try_again">Try again</string>
+
+    <string name="send_data_body">In case of positive COVID-19 test you will be contacted by worker of hygienic station. You will be sent SMS message containing verification code for sneding data.</string>
+    <string name="send_data_verification_hint">Verification code</string>
+    <string name="send_data_button_text">Verify and send data</string>
+    <string name="send_data_code_invalid">Verification code is invalid</string>
+    <string name="send_data_code_expired_header">Verification code is expired</string>
+    <string name="send_data_code_expired_body">Request new SMS message with new verification code from hygienic station administration.</string>
+    <string name="send_data_failure_header">We couldn\'t send your data</string>
+    <string name="send_data_failure_body">Check your internet connection and try again.</string>
+    <string name="send_data_back">Back</string>
+    <string name="send_data_close">Close</string>
+    <string name="send_data_try_again">Try again</string>
+    <string name="send_data_success_header">Data were successfully sent</string>
+    <string name="send_data_success_body_1">Thank you for fighting against the spread of coronavirus.</string>
+    <string name="send_data_success_body_2">Please cooperate with workers of hygienic station on searching all people you were in contact with.</string>
```

Comment:
`hygienic station` v angličtině neexistuje, znamená to "čistá stanice". Používáme překlad "public health office"